doc/getting_started.rst

```diff
@@ -0,0 +1,230 @@
+Getting Started
+===============
+
+The purpose of this guide is to illustrate some of the main features that
+``scikit-learn`` provides. It assumes a very basic working knowledge of
+machine learning practices (model fitting, predicting, cross-validation,
+etc.). Please refer to our :ref:`installation instructions
+<installation-instructions>` for installing ``scikit-learn``.
+
+``Scikit-learn`` is an open source machine learning library that supports
+supervised and unsupervised learning. It also provides various tools for
+model fitting, data preprocessing, model selection and evaluation, and many
+other utilities.
+
+Fitting and predicting: estimator basics
+----------------------------------------
+
+``Scikit-learn`` provides dozens of built-in machine learning algorithms and
+models, called :term:`estimators`. Each estimator can be fitted to some data
+using its :term:`fit` method.
+
+Here is a simple example where we fit a
+:class:`~sklearn.ensemble.RandomForestClassifier` to some very basic data::
+
+  >>> from sklearn.ensemble import RandomForestClassifier
+  >>> clf = RandomForestClassifier(random_state=0)
+  >>> X = [[ 1,  2,  3],  # 2 samples, 3 features
+  ...      [11, 12, 13]]
+  >>> y = [0, 1]  # classes of each sample
+  >>> clf.fit(X, y)
+  RandomForestClassifier(random_state=0)
+
+The :term:`fit` method generally accepts 2 inputs:
+
+- The samples matrix (or design matrix) :term:`X`. The size of ``X``
+  is typically ``(n_samples, n_features)``, which means that samples are
+  represented as rows and features are represented as columns.
+- The target values :term:`y` which are real numbers for regression tasks, or
+  integers for classification (they can also be strings). For unsupervized
```

Comment:
```suggestion
  integers for classification (or any other discrete set of values). For unsupervized
```

my point is not really to be accurate, I just don't want us to be misleading the user here.